pkg/client/container.go

```diff
@@ -77,9 +95,30 @@ func (c Client) DeleteContainer(ctx context.Context, id *container.ID, opts ...C
 }
 
 func (c Client) GetEACL(ctx context.Context, id *container.ID, opts ...CallOption) (*eacl.Table, error) {
+	v, err := c.getEACL(ctx, id, true, opts...)
+	if err != nil {
+		return nil, err
+	}
+
+	return v.table, nil
+}
+
+func (c Client) GetEACLWithSignature(ctx context.Context, id *container.ID, opts ...CallOption) (*EACLWithSignature, error) {
+	return c.getEACL(ctx, id, false, opts...)
+}
+
+func (c Client) getEACL(ctx context.Context, id *container.ID, verify bool, opts ...CallOption) (*EACLWithSignature, error) {
```

Comment:
I would name this private function `getEACLWithSignature`. A bit long, though.

Reply:
I think more short name is better here since method is private and it does not have variations.